safehttp/plugins/cors/cors.go

```diff
@@ -154,6 +154,7 @@ func (it *Interceptor) Before(w *safehttp.ResponseWriter, r *safehttp.IncomingRe
 	return safehttp.Result{}
 }
 
+// Commit is a no-op required to satisfy the safehttp.Interceptor interface.
```

Comment:
nit: the comma actually changes the meaning of the sentence.

Reply:
Right, my bad